File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
    int a = 0;
    int b = 1;
    while(true){
        if (b==nums.size()) break;
        if (nums[a] == nums[b]){
            nums.erase(nums.begin()+b);
        } else{
            a++;
            b++;
        }
    }
    return nums.size();
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        j = 1
        uniqueCount = 1
        while (j < len(nums)):
            numI = nums[i]
            numJ = nums[j]
            j += 1
            if numI == numJ:
                continue
            nums[uniqueCount] = nums[j-1]
            uniqueCount += 1
            i = j-1
        return uniqueCount